angular.json

```diff
@@ -61,6 +61,7 @@
               "node_modules/bpmn-js/dist/assets/bpmn-font/css/bpmn-codes.css",
               "node_modules/bpmn-js/dist/assets/bpmn-font/css/bpmn-embedded.css",
               "node_modules/bpmn-js-properties-panel/dist/assets/bpmn-js-properties-panel.css",
+              "src/ng-multiselect-dropdown.scss",
```

Comment:
This file should not be included in the implementation, but in the library itself. You can add it as an asset in ng-package.json, the import line should then be similar to the import line below (app.css from layout lib). Then, when this is released, an instruction to the release notes should be added that this scss file should be added to the implementation's angular.json.

Reply:
I could also include it from within the node_modules like stated above

Reply:
No because the libs are built to the dist folder in this project